### PR TITLE
MB-5337 Add Camp LeJeune transportation office with USMC GBLOC 

### PR DESCRIPTION
**Pull request by Ryan-Koch**
## Description

Migration that inserts a new transportation office to handle all USMC moves, which is given the USMC GBLOC value. It's basically a copy of `DMO Camp LeJeune` but with the USMC BLOC.

## Setup

Run the migration step below locally, once done you should see the entry in your local dev db.

```sh
make db_dev_migrate
```

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

### CODE REVIEW

File: migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+--
+-- Data for Name: transportation_offices; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+INSERT INTO public.transportation_offices VALUES ('ccf50409-9d03-4cac-a931-580649f1647a', 'aa899628-dabb-4724-8e4a-b4579c1550e0', 'Camp LeJeune (USMC)', '0f631e20-4edf-457d-8de8-431b4e17f5ae', 34.668,-77.327866, 'Monday; Tuesday; Wednesday and Friday 0730 -1600; Thursday 0730 - 1500', 'Walk-In Help; Briefings; Computer Lab; Appointments; QA Inspections', NULL, '2020-11-10 17:47:34.148582', '2020-11-10 17:47:34.148582', 'USMC');
+--
+-- Data for Name: office_emails; Type: TABLE DATA; Schema: public; Owner: postgres
```
Comment (duncan-truss) [inline migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql:6]: I see an office_emails record for the `DMO Camp LeJune` transportation office too did you want to add that as well

Comment (monfresh) [reply migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql:6]: Are we sure this USMC entry is supposed to have the same address as the regular DMO Camp LeJeune?

Comment (monfresh) [reply migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql:6]: Same question for the hours and services. Are they the same? Does it matter?

File: migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+--
+INSERT INTO public.office_phone_lines VALUES ('c20bbee8-72a8-4510-9dcb-20757e05ad36', 'ccf50409-9d03-4cac-a931-580649f1647a', '(910) 451-2377', NULL, false, 'voice', '2020-11-10 17:47:34.148582', '2020-11-10 17:47:34.148582');
+INSERT INTO public.office_phone_lines VALUES ('3f677f42-5191-4548-b50c-b94a3a48d99c', 'ccf50409-9d03-4cac-a931-580649f1647a', '750-4377', NULL, true, 'voice', '2020-11-10 17:47:34.148582', '2020-11-10 17:47:34.148582');
+
```
Comment (monfresh) [inline migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql:10]: It's interesting that the `number` column in `office_phone_lines` is freeform and not enforcing any kind of standard. I would expect at least an area code for each number.

File: migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql (hunk 1 above)
Comment (Ryan-Koch) [reply migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql:6]: I believe we're using the same data for this as the DMO version (that's what my read of the story anyway). I'll ping Alexi in slack to verify. Also good catch on noticing I missed the email record.

File: migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql (hunk 2 above)
Comment (Ryan-Koch) [reply migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql:10]: Yeah I thought that was a bit off too

File: migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql (hunk 1 above)
Comment (Ryan-Koch) [reply migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql]: Added in the email record and got confirmation from Alexi that the data should be the same as the DMO record.

**Review by duncan-truss — APPROVED** [review]

**Review by reggieriser — CHANGES_REQUESTED** [review]
I ran the migration and it added the records as expected.  Just asking for one small change though -- can we use `now()` instead of the hard-coded timestamps?

BTW, I assume the 7-digit phone number is because that's marked as a DSN line where area code doesn't apply.

File: migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
+--
+-- Data for Name: transportation_offices; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+INSERT INTO public.transportation_offices VALUES ('ccf50409-9d03-4cac-a931-580649f1647a', 'aa899628-dabb-4724-8e4a-b4579c1550e0', 'Camp LeJeune (USMC)', '0f631e20-4edf-457d-8de8-431b4e17f5ae', 34.668,-77.327866, 'Monday; Tuesday; Wednesday and Friday 0730 -1600; Thursday 0730 - 1500', 'Walk-In Help; Briefings; Computer Lab; Appointments; QA Inspections', NULL, '2020-11-10 17:47:34.148582', '2020-11-10 17:47:34.148582', 'USMC');
+--
+-- Data for Name: office_phone_lines; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+INSERT INTO public.office_phone_lines VALUES ('c20bbee8-72a8-4510-9dcb-20757e05ad36', 'ccf50409-9d03-4cac-a931-580649f1647a', '(910) 451-2377', NULL, false, 'voice', '2020-11-10 17:47:34.148582', '2020-11-10 17:47:34.148582');
```
[the remark this answers is not in the document]
Comment (Ryan-Koch) [reply migrations/app/schema/20201110162713_make_usmc_transportation_office.up.sql]: Done 👍

**Review by reggieriser — APPROVED** [review]
Thanks for making the change.  LGTM!  🚀

**Pull request merged**